.github/workflows/docker-build.yaml

```
@@ -74,6 +78,7 @@ jobs:
       uses: docker/build-push-action@v2
```

Given the error:
```
Warning: Unexpected input(s) 'provenance', valid inputs are ['add-hosts', 'allow', 'build-args', 'build-contexts', 'builder', 'cache-from', 'cache-to', 'cgroup-parent', 'context', 'file', 'labels', 'load', 'network', 'no-cache', 'outputs', 'platforms', 'pull', 'push', 'secrets', 'secret-files', 'shm-size', 'ssh', 'tags', 'target', 'ulimit', 'github-token']
```
let's bump this action.

```suggestion
      uses: docker/build-push-action@v3
```